src/gui/EditWidgetIcons.h

```diff
@@ -77,7 +77,6 @@ private slots:
     void startFetchFavicon(const QUrl& url);
     void fetchFinished();
     void fetchReadyRead();
-    void fetchCanceled();
```

Comment:
As noted in commit message, this private slot was actually unused since the favicon download progress bar was removed.